lib/pgp.ml

```diff
@@ -1,5 +1,6 @@
-let ( >>= ) = Result.bind
+module Cstruct = Safe_cstruct
```

Comment:
It's kind of OK but the code can be harder to understand because of the shadowing.

Also, I'm not sure "safe" is the right adjective here. I wouldn't qualify these new functions as safer than the original ones, especially since they can hide programming mistakes.